deploy/charts/istio-csr/values.yaml

```diff
@@ -358,3 +358,22 @@ commonLabels: {}
 #     metadata:
 #       name: '{{ template "cert-manager-istio-csr.fullname" . }}-extra-configmap'
 extraObjects: []
+
+# Configures a disruption budget for istio-csr.
+podDisruptionBudget:
+  # Enable or disable the PodDisruptionBudget resource for istio-csr.
+  enabled: false
+
+  # This configures the minimum available pods for disruptions. It can either be set to
+  # an integer (e.g., 1) or a percentage value (e.g., 25%).
+  # It cannot be used if `maxUnavailable` is set.
+  # +docs:property
+  # +docs:type=unknown
+  minAvailable: 1
```

Comment:
This is not a good default value considering the default for replicas (1). It will block an admin from draining the node.

Reply:
If a value is set it renders in the resource which would conflict with the default value of `maxUnavailable`?
So probably best to leave out I think.